pkg/tfvars/libvirt/libvirt.go

```diff
@@ -18,16 +18,16 @@ import (
 )
 
 type config struct {
-	URI             string            `json:"libvirt_uri,omitempty"`
-	Image           string            `json:"os_image,omitempty"`
-	IfName          string            `json:"libvirt_network_if"`
-	MasterIPs       []string          `json:"libvirt_master_ips,omitempty"`
-	BootstrapIP     string            `json:"libvirt_bootstrap_ip,omitempty"`
-	MasterMemory    string            `json:"libvirt_master_memory,omitempty"`
-	MasterVcpu      string            `json:"libvirt_master_vcpu,omitempty"`
-	BootstrapMemory int               `json:"libvirt_bootstrap_memory,omitempty"`
-	MasterDiskSize  string            `json:"libvirt_master_size,omitempty"`
-	DnsmasqOptions  map[string]string `json:"libvirt_dnsmasq_options,omitempty"`
+	URI             string     `json:"libvirt_uri,omitempty"`
+	Image           string     `json:"os_image,omitempty"`
+	IfName          string     `json:"libvirt_network_if"`
+	MasterIPs       []string   `json:"libvirt_master_ips,omitempty"`
+	BootstrapIP     string     `json:"libvirt_bootstrap_ip,omitempty"`
+	MasterMemory    string     `json:"libvirt_master_memory,omitempty"`
+	MasterVcpu      string     `json:"libvirt_master_vcpu,omitempty"`
+	BootstrapMemory int        `json:"libvirt_bootstrap_memory,omitempty"`
+	MasterDiskSize  string     `json:"libvirt_master_size,omitempty"`
+	DnsmasqOptions  [][]string `json:"libvirt_dnsmasq_options,omitempty"`
```

Comment:
The way this is done elsewhere is using a `[]map[string]string`.
https://github.com/openshift/installer/blob/d0462d8b5074448e1917da7f0a5d7a904bd60359/pkg/tfvars/baremetal/baremetal.go#L24

Reply:
ah..interesting..but do you have any objections with the way it is now or do you want to change it to be consistent ?

Reply:
@staebler not urgent by any means - but are you ok with this approach or do you want this to be changed?

Reply:
This is code that you own, so I won't block it the way that you have it. With that being said, here are my arguments for changing it.
1) Consistency in the code so that it is easier to move from one platform to another as a developer.
2) Easier to read the terraform code since the values are read by key rather than by index. With reading by index, a developer reading the code would need to switch over to a far-away file in the repo to see what is stored in each index.

Reply:
makes sense. changed the code to make it a list of mappings. thanks @staebler .